_docs_use-cases/use-case-consumption-change-default-repositoryattribute.md

```diff
@@ -36,3 +36,26 @@ require:
 required_by:
 # End automatic generation
 ---
+
+{{properties.description}}
+
+{% include properties_list.html %}
+
+For every [IdentityAttribute value type]({% link _docs_integrate/attribute-values.md %}#identity-attributes) exactly one [RepositoryAttribute]({% link _docs_integrate/data-model-overview.md %}#localattribute) has the property `isDefault` set, given that at least one RepositoryAttribute of that value type exists.
```

Comment:
This is only true if setting default RepositoryAttributes is not disabled for this Account (which is _by default_ the case for Connectors).